File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int n) {
        int r;
        int sum = 0;
        int temp = n;
        while(n>0){
            r = n%10;
            sum =(sum*10)+r;
            n = n/10;
        }
        if(sum == temp) return true;
        else return false;
    }
}

File: 9-palindrome-number/palindrome-number.c
bool isPalindrome(int x) {
     long a=x,n=0,s;
     if(x<0) return false;
    if(x/10==0) return true;
    while(x>0){
        s=x%10;
        n=n*10+s;
        x=x/10;
    }
    if(n==a) return true;
    else return false;
}